### PR TITLE
feat: Add sample front end helm chart 

### PR DESCRIPTION
**Pull request by ishaansehgal99**
**Reason for Change**:
This helm chart provides a straightforward example for deploying a sample UI on top of a kaito workspace.

### CODE REVIEW

File: charts/frontend/chainlit/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/bin/sh"]
+          args:
+            - -c
+            - |
+              mkdir -p /app/frontend && \
+              pip install chainlit requests && \
+              wget -O /app/frontend/chainlit.py https://raw.githubusercontent.com/Azure/kaito/main/presets/inference/frontend/chainlit.py && \
```
Comment (ishaansehgal99) [inline charts/frontend/chainlit/templates/deployment.yaml:42]: Valid path once merged - tested locally using branch path

**Review by Fei-Guo — COMMENTED** [review]
Remove chainlit unless necessary.

File: presets/test/manifests/frontend/frontend.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
```
Comment (Fei-Guo) [inline presets/test/manifests/frontend/frontend.yaml:1]: We should remove this file.

File: charts/frontend/chainlit/.helmignore [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
[the remark this answers is not in the document]
Comment (ishaansehgal99) [reply charts/frontend/chainlit/.helmignore]: Done

File: charts/frontend/chainlit/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# KAITO Sample Frontend (Chainlit) Helm Chart
+## Install
```
[the remark this answers is not in the document]
Comment (ishaansehgal99) [reply charts/frontend/chainlit/README.md]: Done

File: presets/test/manifests/frontend/frontend.yaml (hunk 2 above)
Comment (ishaansehgal99) [reply presets/test/manifests/frontend/frontend.yaml]: Done

File: demo/inferenceUI/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+   ```
+    env:
+        workspaceServiceURL: "http://<CLUSTER_IP>:80/chat"
+   ```
+   Then deploy with custom values file:
+    ```
+    helm install inference-frontend ./charts/DemoUI/inference -f values.override.yaml
+    ```
+
+Replace `<CLUSTER_IP>` with the IP address of your Kubernetes cluster.
```
Comment (Fei-Guo) [inline demo/inferenceUI/README.md:35]: "the IP address of your Kubernetes cluster." ?    How can user get it?

File: demo/inferenceUI/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+The KAITO InferenceUI Demo provides a sample front-end application that demonstrates
+how to interface with the KAITO Workspace for inference tasks. 
+This guide covers deploying the front-end as a Helm chart in a Kubernetes environment 
+as well as how to run the Python application independently.
+
+### Prerequisites
+
+- Kubernetes cluster with Helm installed (if using helm chart)
+- Python environment (if running the Python file directly)
+- Access to the KAITO Workspace Service endpoint
```
Comment (Fei-Guo) [inline demo/inferenceUI/README.md:12]: "the IP address of your Kubernetes cluster."  what does it mean?

File: demo/inferenceUI/README.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+## Running the Python File Directly
+1. Setup the environment \
+   `pip install chainlit requests`
+2. Download the Python file \
+   `wget -O inference.py https://raw.githubusercontent.com/Azure/kaito/main/demo/inferenceUI/chainlit.py`
+3. Run the application (Replace `<URL_TO_KAITO_WORKSPACE>` with the URL to your Kaito Workspace) \
+   `WORKSPACE_SERVICE_URL=<URL_TO_KAITO_WORKSPACE> chainlit run inference.py -w`
+
+---
+
```
Comment (Fei-Guo) [inline demo/inferenceUI/README.md:48]: You did not mention the step of running an url in web browser?  The use of pod forwardng etc?

File: demo/inferenceUI/README.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+The KAITO InferenceUI Demo provides a sample front-end application that demonstrates
+how to interface with the KAITO Workspace for inference tasks. 
+This guide covers deploying the front-end as a Helm chart in a Kubernetes environment 
+as well as how to run the Python application independently.
+
+### Prerequisites
+
+- Kubernetes cluster with Helm installed (if using helm chart)
+- Python environment (if running the Python file directly)
```
Comment (Fei-Guo) [inline demo/inferenceUI/README.md:11]: We may remove the option of running python file directly.

File: demo/inferenceUI/README.md (hunk 5 above)
Comment (ishaansehgal99) [reply demo/inferenceUI/README.md]: added section

File: demo/inferenceUI/README.md (hunk 7 above)
Comment (ishaansehgal99) [reply demo/inferenceUI/README.md:59]: added section

File: demo/inferenceUI/README.md (hunk 8 above)
Comment (ishaansehgal99) [reply demo/inferenceUI/README.md]: removed

File: demo/inferenceUI/README.md (hunk 6 above)
Comment (ishaansehgal99) [reply demo/inferenceUI/README.md:11]: added section

**Review by Fei-Guo — APPROVED** [review]

**Pull request merged**